eclair-node/src/main/scala/fr/acinq/eclair/api/handlers/Payment.scala

```diff
@@ -38,16 +38,16 @@ trait Payment {
   }
 
   val payInvoice: Route = postRequest("payinvoice") { implicit t =>
-    formFields(invoiceFormParam, amountMsatFormParam.?, "maxAttempts".as[Int].?, "feeThresholdSat".as[Satoshi].?, "maxFeePct".as[Double].?, "externalId".?, "blocking".as[Boolean].?) {
-      case (invoice@PaymentRequest(_, Some(amount), _, nodeId, _, _), None, maxAttempts, feeThresholdSat_opt, maxFeePct_opt, externalId_opt, blocking_opt) =>
+    formFields(invoiceFormParam, amountMsatFormParam.?, "maxAttempts".as[Int].?, "feeThresholdSat".as[Satoshi].?, "maxFeePct".as[Double].?, "externalId".?, "blocking".as[Boolean].?, "pathFindingExperimentName".?) {
```

Comment:
nit: I'm realizing now that since `maxFeePct` is a misleading name (we expect `0.05` if the user wants to set if to 5%) we should verify here that it is between `0.0` and `1.0`. This is completely unrelated to this PR though, I can fix that in a separate follow-up PR if you want.

Reply:
I've changed it so that `maxFeePct` is treated as a percentage as the name implies, so `5` would actually be 5%, which is `0.05`.

Reply:
Can we add the `includeLocalChannelCost` param so we can manually test in a trampoline setting (which is our primary use case for route computation)?

I think we need to rename `feeThresholdSat` and `maxFeePct` to be consistent with `reference.conf`.

Reply:
> I've changed it so that maxFeePct is treated as a percentage as the name implies, so 5 would actually be 5%, which is 0.05.

Thanks. I added this to my note of breaking changes that we really need to document in the next release!

Reply:
> Can we add the includeLocalChannelCost param so we can manually test in a trampoline setting (which is our primary use case for route computation)?

Done in #1942